### ISSUE TITLE
Title Editor needs more work: unable to change Title size,style and effect (underline).

### ISSUE THREAD

**Issue by tezcode** [OpenShot/openshot-qt#1830]
**Title tool not working properly**
When I add title to my project. I click on "Change Font button", I select my desired options, I get the preview of it in the "Sample" window but when I click OK but, it remained unchanged.

Usually the style changes, But size and effects don't. 
When I again click on "Change Font button", the selected options goes back to default.

**System Details**
- Windows 10
- OpenShot Version 2.4.2

**To Reproduce**
1. Go to 'Title'
2. Click on 'Title'
3. Select 'camera border' (as I am using it now).
4. Click on "Change Font" and do some changes to the text (Size and effect).

**Expected behavior**
I wanted the title tool to change the title's characteristics (Style,Size & Effects) according to my wish. And have selected options selected even when I got back to edit the same titles.

**Additional context**
Basically the whole title tool has to be remastered, to save some the user/editor from headaches.

Overall the application is **AWESOME**, but please fix these issues. 

Thank you.

**Comment by peanutbutterandcrackers** [comment]
I can confirm this issue.

@N3WWN and @DylanC - Perhaps this will be of some interest to you two, too?

**Comment by N3WWN** [comment]
@peanutbutterandcrackers - I can confirm that the title editor needs a LOT of work.  I actually use `vi` and hand modify my SVGs so the titles display how I want them (in addition to using resvg 😜 )

**Comment by peanutbutterandcrackers** [comment]
@N3WWN - Hmm... I see. Hopefully the title editor will get some attention too. :+1: lol That moment when hand-modifying is easier than using the tool to facilitate the same work...

**Comment by Hunteil** [comment]
idk if this helps. I've found that sometimes the titles will get confused in the project file preview area. The reason was b/c I already had a file named the same as the another existing title. When I created a new title, it would just replace the old file, but the file would show up in the project files still like normal... in the tracks when you press play, it would show the other file but the thumbnail was still the original... REALLY confusing. Maybe separate issue tho. Haven't had a chance to troubleshot it.

And yea I use Incscape on all my titles... I use templates rarely b/c it just doesn't do enough. But I use them in incscape as a start point.
